srv/salt/ceph/tests/restart/setup.sls

```diff
@@ -0,0 +1,6 @@
+
+update {{ service }}.conf:
+  cmd.run:
+    - name: "date +\\#\\ %c > /srv/salt/ceph/configuration/files/ceph.conf.d/{{ service }}.conf"
```

Comment:
This is destructive of any custom config that's already there, which is fine of course for automated testing, but might be irritating when running smoketests in random dev environments. Should we document somewhere what tests are destructive of?

Reply:
I have not decided where to put this officially.  Maybe in the README of the smoketests?  I think it's fair to state that these tests can be destructive, but will still try to restore the environment to some degree.  The target audience is developers and automated testing.

Reply:
README of the smoketests sounds fine to me.